### DISCUSSION TITLE
Show images in kanban cards (Project Beta)

### DISCUSSION THREAD

**Discussion by martinsangolt** [community/community#5883] [Projects and Issues]
In order to quickly grasp what issues are about — it would be fantastic to be able to display images on the kanban view on the different cards. Much like Trello does today.

**Comment by rmcsharry** [comment]
> In order to quickly grasp what issues are about — it would be fantastic to be able to display images on the kanban view on the different cards. Much like Trello does today.

![image](https://user-images.githubusercontent.com/1156100/162616213-669ad557-b43c-437d-a629-5a225d521b24.png)
